src/atoms/ListItem/index.js

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components';
+import {
+  alignItems,
+  border,
+  borderColor,
+  color,
+  display,
+  flexDirection,
+  flexWrap,
+  justifyContent,
+  order,
+  space,
+  width,
+} from 'styled-system';
+
+// lineHeight is inconsistent
+const footer = (props) => {
+  if (props.footer) {
+    return {
+      display: 'block',
+      paddingRight: `${props.theme.space[1]}px`,
+      paddingBottom: `${props.theme.space[2]}px`,
+      fontSize: `${props.theme.fontSizes[1]}px`,
+      letterSpacing: `${props.theme.letterSpacings.text}`,
+      lineHeight: '1.14',
```

Comment:
1. use the nearest lineHeight in theme.js
2. Rather than have a prop like this it might be better to make another component named appropriately

Reply:
2. for all cases